rock-paper-scissors/scripts/rock-paper-scissors.js

```diff
@@ -21,89 +41,110 @@ function playGame(playerSelection, computerSelection) {
     scissors(computerSelection);
   }
 }
-function computerPlay() {
-  let rPSS = ["rock", "paper", "scissors"];
-  let computerSelection = rPSS[Math.floor(Math.random() * rPSS.length)];
-  return computerSelection;
-}
-
-function roundCounter(roundCount) {}
 
 function rock(computerSelection) {
   if (computerSelection === "rock") {
-    console.log("This round was a draw! Try again!");
+    console.log(`This round was a draw! Try again!`);
     roundCount += 1;
+    nobodyWins += 1;
   } else if (computerSelection === "paper") {
-    console.log(`You Lose! ${computerSelection} beats Rock!`);
-    computerWins += 1;
-    roundCount += 1;
-  } else if (computerSelection === "scissors") {
     console.log(`You win! ${playerSelection} beats ${computerSelection}`);
+    roundCount += 1;
     playerWins += 1;
+  } else if (computerSelection === "scissors") {
+    console.log(`You Lose! ${computerSelection} beats ${playerSelection}!`);
     roundCount += 1;
+    computerWins += 1;
   }
   return roundCounter(roundCount);
 }
 
 function paper(computerSelection) {
   if (computerSelection === "rock") {
-    console.log(`You win! ${playerSelection} beats ${computerSelection}`);
+    console.log(`You Lose! ${computerSelection} beats ${playerSelection}!`);
     roundCount += 1;
+    computerWins += 1;
   } else if (computerSelection === "paper") {
     console.log(`This round was a draw! Try again!`);
-    computerWins += 1;
     roundCount += 1;
+    nobodyWins += 1;
   } else if (computerSelection === "scissors") {
-    console.log(`You lose! ${computerSelection} beats ${playerSelection}`);
-    playerWins += 1;
+    console.log(`You win! ${playerSelection} beats ${computerSelection}`);
     roundCount += 1;
+    playerWins += 1;
   }
   return roundCounter(roundCount);
 }
 
 function scissors(computerSelection) {
   if (computerSelection === "rock") {
-    console.log(`You win! ${playerSelection} beats ${computerSelection}`);
+    console.log(`You Lose! ${computerSelection} beats ${playerSelection}!`);
     roundCount += 1;
-  } else if (computerSelection === "paper") {
-    console.log(`You lose! ${computerSelection} beats ${playerSelection}`);
     computerWins += 1;
-    roundCount += 1;
-  } else if (computerSelection === "scissors") {
+  } else if (computerSelection === "paper") {
     console.log(`You win! ${playerSelection} beats ${computerSelection}`);
+    roundCount += 1;
     playerWins += 1;
+  } else if (computerSelection === "scissors") {
+    console.log(`This round was a draw! Try again!`);
     roundCount += 1;
+    nobodyWins += 1;
   }
   return roundCounter(roundCount);
 }
 
-function roundCOunter(roundCount) {
```

Comment:
Using the lessons from above, try to optimise this function to remove repetition (use lots of functions!).